### PR TITLE
numa_topology: skip dies related tests on aarch64

### PR DESCRIPTION
**Pull request by hs0210**
dies is not supported on arm cpu topology

### CODE REVIEW

File: libvirt/tests/cfg/numa/guest_numa_topology/numa_topology_with_cpu_topology.cfg [hunk 1]
```diff
@@ -8,11 +8,13 @@
     pattern_dmesg_numa_node1 = "ACPI: SRAT: Node 1 PXM 1.*mem.*-0x7fffffff"
     variants:
         - vcpu_with_order:
+            no aarch64
```
Comment (dzhengfy) [inline libvirt/tests/cfg/numa/guest_numa_topology/numa_topology_with_cpu_topology.cfg:11]: Move this above line 9?

File: libvirt/tests/cfg/numa/guest_numa_topology/numa_topology_with_cpu_topology.cfg [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
             cpu_topology = {'dies': '2', 'sockets': '2', 'cores': '2', 'threads': '2'}
             numa_cell_0 = {'id': '0', 'memory': '1048576', 'cpus': '0-7'}
             numa_cell_1 = {'id': '1', 'memory': '1048576', 'cpus': '8-15'}
             numa_cells = [${numa_cell_0}, ${numa_cell_1}]
         - vcpu_without_order:
+            no aarch64
```
Comment (dzhengfy) [inline libvirt/tests/cfg/numa/guest_numa_topology/numa_topology_with_cpu_topology.cfg:17]: Same as above

File: libvirt/tests/cfg/numa/guest_numa_topology/numa_topology_with_cpu_topology.cfg (hunk 1 above)
Comment (hs0210) [reply libvirt/tests/cfg/numa/guest_numa_topology/numa_topology_with_cpu_topology.cfg]: Done.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**